### PR TITLE
[CMSP-918] Drupal 7.100 release

### PR DESCRIPTION
**Pull request by pwtyler**
## Summary
- Adds changelog record for Drupal 7.100

**[Pantheon Release Notes](https://docs.pantheon.io/release-notes)** - New release note for D7.100

**Release**:
- [x] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/releasenotes/2024-03-08-drupal-7-100.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Drupal 7.100 Updates
+published_date: "2024-01-31"
+categories: [drupal]
+---
+
+The latest version of Drupal 7, [7.100](https://www.drupal.org/project/drupal/releases/7.100), became available on Pantheon as of March 8, 20204.
```
Comment (pwtyler) [inline source/releasenotes/2024-03-08-drupal-7-100.md:7]:
```suggestion
The latest version of Drupal 7, [7.100](https://www.drupal.org/project/drupal/releases/7.100), became available on Pantheon as of March 8, 2024.
```

**Comment by pwtyler** [comment]
Link direct to changelog entry:
https://pr-8885-documentation.appa.pantheon.site/release-notes/2024/03/drupal-7-100

File: source/releasenotes/2024-03-08-drupal-7-100.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+categories: [drupal]
+---
+
+The latest version of Drupal 7, [7.100](https://www.drupal.org/project/drupal/releases/7.100), became available on Pantheon as of March 8, 2024.
+
+<h3>Highlights</h3>
+
+* **Announcements Module** 7.100 includes the Annoucements module backported to Drupal 7. Please read the [change record](https://www.drupal.org/node/3424912) _before updating_ if you do not want the module to be automatically enabled.
+* **PHP 8 Compatibility** Bundled with Drupal 7's release is [an update](https://github.com/pantheon-systems/drops-7/pull/201) to Pantheon's Apache Solr module that fixes deprecation warnings with PHP 8.1+.
+* **Bug fixes and security hardening** In addition to the Announcements Module, the release includes a few bug fixes and feature improvements. No security fixes are included in this release. For full details, see the release notes [on Drupal.org](https://www.drupal.org/project/drupal/releases/7.100).
```
Comment (pwtyler) [inline source/releasenotes/2024-03-08-drupal-7-100.md:13]:
```suggestion
* **Bug fixes and security hardening** In addition to the Announcements module, the release includes a few bug fixes and feature improvements. No security fixes are included in this release. For full details, see the release notes [on Drupal.org](https://www.drupal.org/project/drupal/releases/7.100).
```

**Review by stevector — CHANGES_REQUESTED** [review]
We need clearer language than
> Support for Drupal 7 is ending on 5 January 2025.

People will read that and assume that they can't run Drupal 7 on Pantheon after that date which isn't the case.

File: source/releasenotes/2024-03-08-drupal-7-100.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+<h3>Highlights</h3>
+
+* **Announcements Module** 7.100 includes the Annoucements module backported to Drupal 7. Please read the [change record](https://www.drupal.org/node/3424912) _before updating_ if you do not want the module to be automatically enabled.
+* **PHP 8 Compatibility** Bundled with Drupal 7's release is [an update](https://github.com/pantheon-systems/drops-7/pull/201) to Pantheon's Apache Solr module that fixes deprecation warnings with PHP 8.1+.
+* **Bug fixes and security hardening** In addition to the Announcements module, the release includes a few bug fixes and feature improvements. No security fixes are included in this release. For full details, see the release notes [on Drupal.org](https://www.drupal.org/project/drupal/releases/7.100).
+
+
+Upgrade your Drupal 7 site to Drupal 7.100 right from your Pantheon dashboard or Terminus.
+
+Support for Drupal 7 is ending on 5 January 2025.
```
[the remark this answers is not in the document]
Comment (pwtyler) [reply source/releasenotes/2024-03-08-drupal-7-100.md:18]:
```suggestion
```
That's a larger conversation that doesn't need to block getting this out.

**Review by scottbuscemi — APPROVED** [review]

**Review by stevector — APPROVED** [review]
Thanks!

**Pull request merged**